src/argilla/server/search_engine.py

```diff
@@ -191,21 +192,15 @@ async def search(
             "_source": False,
             "query": {"bool": bool_query},
             "sort": ["_score", {"id": "asc"}],
+            "from": offset,
+            "size": limit,
```

Comment:
I see we have all the extra body params as `from`, `size`, `sort`, and `source` defined in the `client.search`. I think we should pass them as method args instead of a raw body.
```suggestion
```